libs/ledgerjs/packages/types-live/src/lnsUpsell.ts

```diff
@@ -2,7 +2,7 @@ export type LldNanoSUpsellBannersConfig = {
   manager: boolean;
   accounts: boolean;
   notification_center: boolean;
-  img: string;
   link: string;
-  "%": number;
+  img?: string;
+  "%"?: number;
```

Comment:
[ASK] I know it's for config but having this kind of key can be problematic no? 
But why don't use something like `discount`?